### PR TITLE
292 submit work

### PR DESCRIPTION
**Pull request by rkzel**
FE ready.

### CODE REVIEW

File: apps/projects/app/components/Panel/SubmitWork/SubmitWork.js [hunk 1]
```diff
@@ -1,19 +1,147 @@
...
+
+  changeField = ({ target: { name, value } }) => this.setState({ [name]: value })
+  setAck1 = () => this.setState(prevState => ({ ack1: !prevState.ack1 }))
+  setAck2 = () => this.setState(prevState => ({ ack2: !prevState.ack2 }))
+
+  onSubmitWork = () => {
+    console.log('Submit', this.state)
+  }
+
+  canSubmit = () => !(this.state.ack1 && this.state.ack2 && this.state.proof)
```
Comment (Quazia) [inline apps/projects/app/components/Panel/SubmitWork/SubmitWork.js]: Can we validate that the hours worked field is populated and a number as well?

File: apps/projects/app/components/Panel/SubmitWork/SubmitWork.js [hunk 2]
```diff
@@ -1,19 +1,147 @@
...
+
+  render() {
+    // TODO: replace with props
+    const account = 'accplaceholder'
```
Comment (Quazia) [inline apps/projects/app/components/Panel/SubmitWork/SubmitWork.js]: Can you explain this part a little bit? Is this the github account information?

File: apps/projects/app/components/Panel/SubmitWork/SubmitWork.js [hunk 3]
```diff
@@ -1,19 +1,147 @@
...
+        />
+        <FormField
+          label="Additional Comments"
+          input={
+            <TextInput.Multiline
+              name='comments'
+              rows={5}
+              style={{ resize: 'none', height: 'auto' }}
+              onChange={this.changeField}
+              placeholder="Do you have any other comments or details you would like to provide that hasn’t already been described elsewhere?"
```
Comment (Quazia) [inline apps/projects/app/components/Panel/SubmitWork/SubmitWork.js]: I would shorten this placeholder, doesn't need to be this long/polite.

File: apps/projects/app/components/Panel/SubmitWork/SubmitWork.js [hunk 4]
```diff
@@ -1,19 +1,147 @@
...
+            />
+          }
+        />
+
+        <AckRow>
+          <div style={{width: '23px'}}>
+            <CheckButton checked={this.state.ack1} onChange={this.setAck1}/>
+          </div>
+          <AckText>
+            I acknowledge that that my work must be accepted for me to receive the payout.
```
Comment (Quazia) [inline apps/projects/app/components/Panel/SubmitWork/SubmitWork.js]: Double that

File: apps/projects/app/components/Panel/SubmitWork/SubmitWork.js (hunk 1 above)
Comment (rkzel) [reply apps/projects/app/components/Panel/SubmitWork/SubmitWork.js]: Done.

File: apps/projects/app/components/Panel/SubmitWork/SubmitWork.js (hunk 3 above)
Comment (rkzel) [reply apps/projects/app/components/Panel/SubmitWork/SubmitWork.js]: Shortened.

File: apps/projects/app/components/Panel/SubmitWork/SubmitWork.js (hunk 4 above)
Comment (rkzel) [reply apps/projects/app/components/Panel/SubmitWork/SubmitWork.js]: Fixed.

File: apps/projects/app/components/Panel/SubmitWork/SubmitWork.js (hunk 2 above)
Comment (rkzel) [reply apps/projects/app/components/Panel/SubmitWork/SubmitWork.js]: Done.

**Pull request merged**